style.css

```diff
@@ -380,7 +384,7 @@ main {
   display: flex;
   justify-content: center;
   align-items: center;
-  background: #f55800;
+  background: #cf4b00;
 }
 
 .button:hover {
```

Comment:
- [ ] Please fix the contrast ratio issue between text color and background color for `See Project` and `Get my resume` buttons at hovering.
You can check your fix using this [site](https://www.aditus.io/button-contrast-checker/).

![image](https://user-images.githubusercontent.com/68796232/166654581-8785969f-cda3-460a-81a3-2fddf0e4f53c.png)
![image](https://user-images.githubusercontent.com/68796232/166654599-6283f5f3-134e-45c3-9c8a-d085c0f67f05.png)

### Please make sure that both team members have pushed commits doing the required changes.